docs/troubleshoot/06_exercise.md

```diff
@@ -27,19 +27,19 @@ nextflow run hello-gatk.nf
 
     ```console title="hello-gatk.nf" linenums="5"
     // Execution environment setup
-    params.baseDir = "/workspace/gitpod/troubleshoot"
-    $baseDir = params.baseDir
+    params.projectDir = "/workspace/gitpod/troubleshoot"
+    $projectDir = params.projectDir
 
     // Primary input
-    params.reads_bam = "${baseDir}/data/samplesheet.csv"
+    params.reads_bam = "${projectDir}/data/samplesheet.csv"
     ```
 
-    However, it can be noted that there is a `$` before `baseDir` on line 7 which would suggest it is a variable.
+    However, it can be noted that there is a `$` before `projectDir` on line 7 which would suggest it is a variable.
```

Comment:
```suggestion
    However, it can be noted that there is a `$` before `baseDir` on line 7 which would suggest it is a variable.
```
```suggestion
    However, it can be noted that there is a `$` before `baseDir` on line 7 which would suggest it is a variable.
```